driver/userspace.c

```diff
@@ -374,21 +374,21 @@ WnbdCreateConnection(PWNBD_EXTENSION DeviceExtension,
     if (!Device->Properties.BlockSize || !Device->Properties.BlockCount ||
         Device->Properties.BlockCount > ULLONG_MAX / Device->Properties.BlockSize)
     {
-        WNBD_LOG_ERROR("Invalid block size or block count. "
-                       "Block size: %d. Block count: %lld.",
-                       Device->Properties.BlockSize,
-                       Device->Properties.BlockCount);
+        WNBD_LOG_WARN("Invalid block size or block count. "
```

Comment:
Let's stick to an error, we failed to create the connection because of that.

Probably a good rule of thumb: if we manage to resume the connection after the error we can use a warning, if not, we should probably use error level.